cli/CHANGELOG.md

```diff
@@ -1,4 +1,47 @@
 <!-- See the ../guides/writing-the-cypress-changelog.md for details on writing the changelog. -->
+## 15.0.0
+
+_Released 07/22/2025 (PENDING)_
```

Comment:
Pending release date for 15.0.0 should be later than 14.5.3 (_Released 7/30/2025 (PENDING)_) or the versions would need to be merged.

Reply:
[`1e96eb1` (#31408)](https://github.com/cypress-io/cypress/pull/31408/commits/1e96eb1de62cca51b3afcaeb264951db2b08960d)